scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -323,6 +325,7 @@ task GetMaxTableIdLegacy {
   }
   output {
       Int max_table_id = read_int(stdout())
+      File gvs_ids = "gvs_ids"
```

Comment:
is there any benefit to making gvs_ids a var?

Reply:
i'm not sure what you mean by a var

Reply:
I think what you have works great--- I was just "thinking out loud"....